tests/testing_config.yaml

```diff
@@ -21,9 +21,9 @@ simulator:
         batch_size: 32
         prob_galaxy: 0.5
         star_flux_min: 1e5
-    background:
-        _target_: bliss.simulator.background.ConstantBackground
-        background: [865.0]
+    # background:
+    #     _target_: bliss.simulator.background.SimulatedSDSSBackground
+    #     sdss_fields: ${simulator.sdss_fields}
```

Comment:
Should these lines be uncommented?

Reply:
They shouldn't be there at all - I forgot to mention this to @jeff-regier earlier but if we want to remove support for ConstantBackground then this isn't needed in the testing config at all. The reason for removing ConstantBackgrounds is that z-scoring a constant background results in NaN's since the standard deviation of the background would just be 0.